core-js/src/main/javascript/cdf-legacy/Dashboards.Main.js

```diff
@@ -697,15 +697,15 @@ Dashboards.init = function(components){
     this.syncParametersInit();
 
   }
-  
+
   if($.isArray(components)) { this.addComponents(components); }
 
   // Now we need to go through all components we have and attach this
-  // initInstance to all 
+  // initInstance to all
   _.chain(Dashboards.components)
-  .where({initInstance:undefined})
+  .filter(function(c) { return c.initInstance === undefined; })
```

Comment:
The new `underscore` version changed the behavior here in some way, causing the `undefined` value `initInstance` property to be ignored, resulting in none of the components to be initialized. A similar fix had already been done in the past to the RequireJS counterpart code, which was already using a more recent underscore version.